crates/uv/src/commands/venv.rs

```diff
@@ -160,12 +163,15 @@ async fn venv_impl(
         .with_options(OptionsBuilder::new().exclude_newer(exclude_newer).build());
 
         // Resolve the seed packages.
+        let version_specifier = VersionSpecifier::from_str("<3.12").unwrap();
+        let mut requirements = vec![Requirement::from_str("pip").unwrap()];
+
+        if version_specifier.contains(interpreter.python_version()) {
```

Comment:
Could you use `interpreter.python_tuple()` instead and just do a numerical comparison instead of using the `VersionSpecifier`? It should be faster and simpler.

Reply:
updated to use numerical comparison